packaging/ubuntu-14.04/snapd.install

```diff
@@ -6,6 +6,7 @@ usr/bin/snap-repair /usr/lib/snapd/
 usr/bin/snap-failure /usr/lib/snapd/
 usr/bin/snap-update-ns /usr/lib/snapd/
 usr/bin/snapd /usr/lib/snapd/
+usr/bin/snap-preseed /usr/lib/snapd/
```

Comment:
wondering if we should wait to do this until we can have a spread test and is (almost) fully working?

Reply:
No problem, makes sense, done.